### ISSUE TITLE
Problems under Android 6 [solved]

### ISSUE THREAD

**Issue by lenzp** [mendhak/gpslogger#743]
GPSLogger works perfektly on a motorola moto G5 unter Android 8. 
On a tablet Lenovo TB3-850M under Android 6 I made exactly the same settings. It can send test e-mails and receive them, but if I send GPX files, I get this error message:
![IMG_2440](https://user-images.githubusercontent.com/50932693/58216060-0c831c80-7cfd-11e9-82c3-0914cbbcedc5.JPG)

**Comment by mendhak** [comment]
Is your SMTP server TLS protected - if it is, have you validated certificate and did that validation work?

If not TLS, then this appears to be trying to handshake with a non-TLS server. 

You can try to toggle the 'Implicit SSL' setting on the Email screen. 

Another thing to try is to [enable debug logging](https://gpslogger.app/#troubleshooting) and then perform the _working_ email test in the preference screen, followed by the attempt to send a file via the main screen.  This should capture some info in the debug log.  You can then have a look or send it to me to look at.

**Comment by lenzp** [comment]
Thank you very much for your comment, I will try your advices in the evening.
But the test message works. Are there any differences between the test e-mail and the regular data e-mails regarding handshakes or TLS?

**Comment by lenzp** [comment]
When I went back from arcor.de to gmail.com, I found that the port 465 was set. I tried some combinations and found out: e-mails are correctly sent for
- port 587 without implicit SSL
- port 465 with implicit SSL
All other combination don't work and I guess, yesterday I had one of the wrong combinations.
Perhaps this is something worth to mention in die docu or in the FAQ.
Thank you very much again for pushing me in the right direction!
